mps-model-adapters/src/main/kotlin/org/modelix/model/mpsadapters/MPSDevKitDependencyAsNode.kt

```diff
@@ -67,7 +67,9 @@ data class MPSDevKitDependencyAsNode(
     }
 
     override fun getPropertyValue(property: IProperty): String? {
-        return if (property.conformsTo(BuiltinLanguages.MPSRepositoryConcepts.LanguageDependency.name)) {
+        return if (property.isIdProperty()) {
+            reference.serialize()
```

Comment:
We have the same issue in `MPSJavaModuleFacetAsNode`, `MPSModuleDependencyAsNode`, `MPSSingleLanguageDependencyAsNode`, `MPSDevKitDependencyAsNode`.
They all are children of `MPSModuleAsNode`.

(You fixed `MPSDevKitDependencyAsNode` and `MPSSingleLanguageDependencyAsNode`.)

You can now test it with Junit like in https://github.com/modelix/modelix.core/blob/main/bulk-model-sync-lib-mps-test/src/test/kotlin/org/modelix/model/sync/bulk/lib/test/ModelImporterWithMPSAdaptersTest.kt

Reply:
`MPSProjectModuleAsNode` and `MPSProjectAsNode` have the issue to. But the are usually not used within `ModelImporeter`.

Reply:
I added it to the concepts you listed.